dnf-behave-tests/fixtures/repos.d/dnf-ci-fifthparty-modular.repo

```diff
@@ -0,0 +1,5 @@
+[dnf-ci-fifthparty-modular]
```

Comment:
> fifthparty

This is how structuring test cases and their data gets out of hand :slightly_frowning_face: